### PR TITLE
Fix MinGW 64bit build.

### PR DESCRIPTION
**Pull request by huseyinkozan**
Hi,

After try to build with Qt 5.12 MinGW 7.3 64, got some errors. This PR fixes them.

### CODE REVIEW

File: src/asmCrashReport.cpp [hunk 1]
```diff
@@ -237,7 +245,12 @@ namespace asmCrashReport
 
       if ( inExceptionInfo->ExceptionRecord->ExceptionCode == EXCEPTION_STACK_OVERFLOW )
       {
+          // https://stackoverflow.com/a/38019482
+#ifdef _M_IX86
          frameInfoList += _addr2line( sProgramName, reinterpret_cast<void*>(inExceptionInfo->ContextRecord->Eip) );
+#else
```
Comment (asmaloney) [inline src/asmCrashReport.cpp]: Could you please use `#elif _M_X64` and make the `#else` produce an `#error` saying this needs to be implemented (as above).

File: src/asmCrashReport.cpp [hunk 2]
```diff
@@ -135,8 +135,16 @@ namespace asmCrashReport
...
       stackFrame.AddrFrame.Offset = context->Ebp;
       stackFrame.AddrFrame.Mode = AddrModeFlat;
+#elif _M_X64
+            image = IMAGE_FILE_MACHINE_AMD64;
+            stackFrame.AddrPC.Offset = context->Rip;
+            stackFrame.AddrPC.Mode = AddrModeFlat;
+            stackFrame.AddrFrame.Offset = context->Rsp;
+            stackFrame.AddrFrame.Mode = AddrModeFlat;
+            stackFrame.AddrStack.Offset = context->Rsp;
+            stackFrame.AddrStack.Mode = AddrModeFlat;
```
Comment (asmaloney) [inline src/asmCrashReport.cpp]: Please fix spacing to match code (indent levels are 3-spaces - no tabs).

File: src/asmCrashReport.cpp [hunk 3]
```diff
@@ -135,8 +135,16 @@ namespace asmCrashReport
...
 #else
-      // see http://theorangeduck.com/page/printing-stack-trace-mingw
+    // see http://theorangeduck.com/page/printing-stack-trace-mingw
```
Comment (asmaloney) [inline src/asmCrashReport.cpp]: Please don't change spacing.

**Review by asmaloney — CHANGES_REQUESTED** [review]
Thanks for the submission. Just a couple of small things to fix up.

File: src/asmCrashReport.cpp (hunk 1 above)
Comment (huseyinkozan) [reply src/asmCrashReport.cpp]: Ok

File: src/asmCrashReport.cpp (hunk 3 above)
Comment (huseyinkozan) [reply src/asmCrashReport.cpp]: Ok

**Comment by asmaloney** [comment]
Thank you for the contribution!

**Pull request merged**